app/src/components/integrations/elements/bts/btsPropertiesForIssueForm/btsPropertiesForIssueForm.jsx

```diff
@@ -455,7 +455,15 @@ export class BtsPropertiesForIssueForm extends Component {
                   customBlockCreator={this.getCustomBLockConfig}
                   customFieldWrapper={IntegrationFormField}
                   defaultOptionValueKey={this.defaultOptionValueKey}
-                />
+                  disabled={disabled}
+                >
+                  <div className={cx('show-hint-wrapper')}>
+                    <span className={cx('show-hint-text')}>
+                      {intl.formatMessage(messages.showFieldsHeader)}
+                    </span>
+                    <ShowWithTooltip />
+                  </div>
```

Comment:
Maybe better decide here should we pass children or not not directly in the component